### DISCUSSION TITLE
Apenas para conseguir a insignia

### DISCUSSION THREAD

**Discussion by LuiVLoureiro** [community/community#114501] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Apenas para conseguir a insiginia

**Comment by LuiVLoureiro** [comment]
Comentario 1 para ser respondido

**Comment by ettaboyle** [comment]
Hi @LuiVLoureiro -- You've done two things in this post that violate our [Community Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md).

1. **This post is very clearly badge hunting.**   We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

2. **Non-English Post**. We recognize that this is not ideal for a global audience, but to help ensure that everyone has a good experience free of antagonism and harassment, we ask that all users post in English. **Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.** We’ll be locking any posts in languages other than English for now, including this one.

3. I can see that you have two other posts that violate our code of conduct.  
- https://github.com/orgs/community/discussions/114504
- https://github.com/orgs/community/discussions/114505
I have locked both of those posts and unmarked the answer.

I am giving you a 7 day ban from  participating in the Community Discussions. Any future violations may result in an indefinite block from the Community. Thanks for understanding.
